2025_12_LDV/README.md

```diff
@@ -0,0 +1,21 @@
+# Consultatie LDV release 2025
+
+## Context
+De standaard Logboek Dataverwerkingen beschrijft een manier om technisch interoperabele functionaliteit voor het loggen van Dataverwerkingen te implementeren, door voor de volgende functionaliteit de interface en het gedrag voor te schrijven:
+
+- het vastleggen van logs van dataverwerkingen
+- het aan elkaar relateren van logs van dataverwerkingen
+- het aan elkaar relateren van dataverwerkingen over de grenzen van systemen
+- Door Dataverwerkingen te loggen volgens de standaard kunnen organisaties het datagebruik verantwoorden.
+
+## Werkingsgebied
+Functioneel toepassingsgebied: De standaard kan worden toegepast bij het uitvoeren van dataverwerkingen en het aan elkaar relateren van dataverwerkingen van verschillende verwerkingsverantwoordelijken.
+
+Organisatorisch werkingsgebied: Nederlandse overheden (Rijk, provincies, gemeenten en waterschappen) en instellingen uit de (semi-) publieke sector.
+
+## Consultatie
+Vanuit Logius vragen wij u om de LDV standaard te consulteren op de volgende twee documenten:
+- [Het normatief document](https://logius-standaarden.github.io/logboek-dataverwerkingen/)
+- [Het juridisch beleidskader](https://logius-standaarden.github.io/logboek-dataverwerkingen-juridisch-beleidskader/)
+
+Voor vragen of opmerkingen kunt u terecht bij api@logius.nl of kunt een issue indienen via [Github](https://github.com/Logius-standaarden/logboek-dataverwerkingen/issues)
```

Comment:
Kunnen we de sectie van https://github.com/Logius-standaarden/Openbare-Consultaties/tree/master/2025-10-CloudEvents#reageren overnemen hier voor consistentie?